py3status/modules/mpd_status.py

```diff
@@ -10,12 +10,12 @@
     hide_when_paused: hide the status if state is paused (default False)
     hide_when_stopped: hide the status if state is stopped (default True)
     host: mpd host (default 'localhost')
-    idle_subsystems: a space seperated string of subsystems to idle on.
+    idle_subsystems: a comma seperated string of subsystems to idle on.
```

Comment:
(Imo) The users could supply the list instead and we would have this module directly pass this off as-is.

Reply:
ohhh, nice! didn't know that was possible! implemented.